From a82fea9663f0c70cf57cd06650f400e878afc5e4 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Fri, 16 Apr 2021 10:48:36 -0700
Subject: [PATCH] Fix host contamination of include files

python3-icu-2.5-r0 do_package_qa: QA Issue: python3-icu: The compile log indicates that host include and/or library paths were used.

Also, don't use icu-config

Upstream-Status: Inappropriate [OE specific]
Signed-off-by: Armin Kuster <akuster808@gmail.com>
Signed-off-by: Khem Raj <raj.khem@gmail.com>

---
 setup.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index d308b80..56e9c74 100644
--- a/setup.py
+++ b/setup.py
@@ -81,7 +81,7 @@ Building PyICU %s for ICU %s (max ICU major version supported: %s)
 
 CONFIGURE_WITH_ICU_CONFIG = {
     'darwin': True,
-    'linux': True,
+    'linux': False,
     'freebsd': False, # not tested
     'win32': False,   # no icu-config
     'sunos5': False,  # not tested
@@ -273,7 +273,6 @@ setup(name="PyICU",
       ext_modules=[Extension('icu._icu_',
                              [filename for filename in sorted(os.listdir(os.curdir))
                               if filename.endswith('.cpp')],
-                             include_dirs=_includes,
                              extra_compile_args=_cflags,
                              extra_link_args=_lflags,
                              libraries=_libraries)],
